src/main/java/org/openstreetmap/atlas/geography/atlas/raw/slicing/RawAtlasSlicer.java

```diff
@@ -153,10 +153,10 @@ protected static boolean fromSameCountry(final Line one, final Line two)
             firstCountries.retainAll(new HashSet<>(secondCountries));
             return !firstCountries.isEmpty();
         }
-
-        throw new CoreException(
```

Comment:
Quick question - why was this taken out?

Reply:
good question-- I found it was causing some issues during the loading of the point-and-line sliced shard data from other countries. but things have changed a lot since then, so I'll test and see if I need this removed still.